examples/nextjs-blog/components/header.js

```diff
@@ -45,17 +45,14 @@ const Header = () => {
         <nav>
           <ul>
             {mainNav.items.map(item => (
-              <li key={item.uuid}>
+              <li key={item.uuid} className={item.cssClass}>
                 <Link route={item.url}>
-                  <a>{item.title}</a>
+                  <a target={item.newWindow ? '_blank' : '_self'}>
+                    {item.title}
+                  </a>
                 </Link>
               </li>
             ))}
-            <li>
```

Comment:
I added this as an example button because of the design. Don't we want it?

Reply:
I added it to the CMS so will still show but just with the button styling

Reply:
cool cool